datasets/igbo_monolingual/README.md

```diff
@@ -0,0 +1,159 @@
+---
+annotations_creators:
+- found
+language_creators:
+- found
+languages:
+- ig
+licenses:
+- unknown
+multilinguality:
+- monolingual
+size_categories:
+  bbc-igbo:
+  - 1K<n<10K
+  eze_goes_to_school:
+  - n<1K
+  igbo-radio:
+  - n<1K
+  jw-books:
+  - n<1K
+  jw-nt-igbo:
+  - n<1K
+  jw-ot-igbo:
+  - n<1K
+  jw-teta:
+  - n<1K
+  jw-ulo_nche:
+  - n<1K
+  jw-ulo_nche_naamu:
+  - n<1K
+source_datasets:
+- original
+task_categories:
+- sequence-modeling
+task_ids:
+- language-modeling
+---
+
+# Dataset Card for Igbo Monolingual Dataset
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-fields)
+  - [Data Splits](#data-splits)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** https://github.com/IgnatiusEzeani/IGBONLP/tree/master/ig_monoling
+- **Repository:** https://github.com/IgnatiusEzeani/IGBONLP/tree/master/ig_monoling
+- **Paper:** https://arxiv.org/abs/2004.00648
+
+### Dataset Summary
+
+[More Information Needed]
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+[More Information Needed]
+
+## Dataset Structure
+
+### Data Instances
+
+[More Information Needed]
+
+### Data Fields
+
+[More Information Needed]
```

Comment:
can you briefly describe the data fields depending on the config ? (content, tags, description etc.)

and also mention that missing `date` values are replaced with empty strings ?